src/EnergyPlus/WindowManager.cc

```diff
@@ -6097,7 +6097,7 @@ namespace WindowManager {
 			//  ( SurfaceWindow(SurfNum)%FrameEmis*(sigma*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin)**4 - rmir) + &
 			//    hcin*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin - tin) )
 
-			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * ( 1.0 + SurfaceWindow( SurfNum ).ProjCorrFrIn ) * ( hcin * ( SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin - tin ) );
+			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * FrameCon * ( SurfaceWindow( SurfNum ).FrameTempSurfOut - SurfaceWindow( SurfNum ).FrameTempSurfIn );
```

Comment:
@mjwitte I see your point. I guess the question is, should the reported values represent the heat transfer through the window, or the heat transfer to the zone air?

Perhaps the best solution is to have two separate report variables: one for convection to the zone (that can be used to compare the sum of the loads as per the original CR), and another for heat transfer through the window (for calculating the U-factor).

I'm ok dropping this change for now, but then the divider heat gain should probably be changed for consistency. The documentation should probably also be adjusted to reflect that the frame heat gain is convection, and not conduction.

What do you think is best?